README.md

```diff
@@ -68,26 +64,25 @@ Lobus exposes all of the classes it uses internally, in case you want to compose
     .addChoices({
       bePaladin: {
         description: 'Become a paladin',
-        effect() {
-          startingClass = 'paladin';
-        },
         prerequisite(choices) {
-          return choices.toughChoice !== 'beBad'
+          return choices.toughChoice === 'beGood'
         }
       }
     },
     {
-      beThief: {
+      thief: {
```

Comment:
For consistency, remove the `be` from the Paladin choice as well, since your comment below also indicates that the property should just be `paladin`.